packages/moonstone/Scrollable/ScrollButton.less

```diff
@@ -0,0 +1,17 @@
+.scrollButton {
+	margin: 0;
+
+	&.small:before {
+		content: none;
+	}
+
+	.bg {
+		will-change: auto;
+		-webkit-transform: initial;
+		transform: initial;
```

Comment:
This removes the transformation and acceleration of the button press animation but doesn't address any of the animation rules. This changes the way the button responds to an out-of-design style.

Reply:
Hi @Djspaceg 

There were some composite layers due to the page controls. I tried to remove them before to enhance list scroll performance.

This PR includes
* the revert of the previous patch to remove the layers and 
* the new patch to remove the layers again.

Without the `transform: initial;` change, there is no way to remove the layers. If my PR breaks the animation rules, I'd like
* to go ahead only to revert of the previous patch and
* not to include the new patch
in this PR. 

Could you let me know your opinion?
Thanks.